src/main/java/uk/gov/cslearning/record/domain/Booking.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.cslearning.record.domain;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+public class Booking {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private int learnerId;
+
+    @Column(nullable = false)
+    private int eventId;
+
+    @Column
+    private String paymentDetails;
+
+    @Column(nullable = false)
+    private String status;
+
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    private LocalDateTime bookingTime;
```

Comment:
Jakub started using `Instant` for this. Can you have a look into what are the pros vs cons of this?